test/metabase/api/database_test.clj

```diff
@@ -153,6 +153,46 @@
       (is (= {:errors {:include "value may be nil, or if non-nil, value must be one of: `tables`, `tables.fields`."}}
              (mt/user-http-request :lucky :get 400 (format "database/%d?include=schemas" (mt/id))))))))
 
+(deftest get-database-usage-info-test
+  (mt/with-temp*
+    [Database [{db-id :id}]
+     Table    [{table-id-1 :id} {:db_id db-id}]
+     Table    [{table-id-2 :id} {:db_id db-id}]
+     ;; question
+     Card     [_                {:database_id db-id
+                                 :table_id table-id-1
+                                 :dataset  false}]
```

Comment:
(not the end of the world, but it might be worth tracking down a vim plugin that aligns map keys like so:

```
{:database_id db-id
 :table_id    table-id-1
 :dataset     false}
```

)

Reply:
yea, I tried to find it but failed to find a good solution. 
maybe I'll go shopping sometimes this weekend again.